Tasks/AzurePowerShell/AzurePowerShell.ps1

```diff
@@ -8,6 +8,8 @@ $scriptInline = Get-VstsInput -Name Inline
 $scriptArguments = Get-VstsInput -Name ScriptArguments
 $targetAzurePs = Get-VstsInput -Name TargetAzurePs
 $customTargetAzurePs = Get-VstsInput -Name CustomTargetAzurePs
+$serviceNameInput = Get-VstsInput -Name ConnectedServiceNameSelector -Default 'ConnectedServiceName'
```

Comment:
Do you want to keep this co-located in the try block, just like how the Azurehelper does.